src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  background-color: #000;
+  margin: 0;
+}
+
+.enclosures-border {
```

Comment:
class - is name of the element - this is not a border, this is a block or box etc.